types/webpack/index.d.ts

```diff
@@ -408,43 +411,35 @@ declare namespace webpack {
         /**
          * Shortcut for resource.exclude
          */
-        exclude?: RuleSetCondition & {
-            [k: string]: any;
-        };
+        exclude?: RuleSetCondition;
         /**
          * Shortcut for resource.include
          */
-        include?: RuleSetCondition & {
-            [k: string]: any;
-        };
+        include?: RuleSetCondition;
         /**
          * Match the issuer of the module (The module pointing to this module)
          */
-        issuer?: RuleSetCondition & {
-            [k: string]: any;
-        };
+        issuer?: RuleSetCondition;
         /**
          * Shortcut for use.loader
          */
-        loader?: RuleSetLoader | RuleSetUse;
+        loader?: RuleSetUse;
```

Comment:
In the schema, it references `ruleSet-loader` too, but it's redundant, since `ruleSet-use` references `ruleSet-use-item`, and `ruleSet-use-item` references `ruleSet-loader`.